hardware/Drivetrain.java

```diff
@@ -199,18 +180,18 @@ public void turn(int degrees, double power) {
         pidRotate.enable();
         if (degrees < 0) {
             while (getAngle() == 0) {
-                setPower(-power, power, -power, power);
+                setPower(new double[] {-power, power, -power, power});
```

Comment:
one line this while loop
in addition to saving lines, it also gives a clearer indication of why we are setting power if the angle is 0; we need to give the robot a little bit of a nudge before the pid can perform